### PR TITLE
Disable FAQ browser on public

### PR DESCRIPTION
**Pull request by imsickofmaps**
Add config flag so menu option only shows if true

### CODE REVIEW

**Comment by imsickofmaps** [comment]
@smn travis seems to be stalled on the queue at the moment, but this is ready for review

File: test/personal.test.js [hunk 1]
```diff
@@ -1843,3 +1844,303 @@ describe("app", function() {
...
+                            'MomConnect. Choose an option:',
+                            '1. Get the full set of messages'
+                        ].join('\n')
+                    })
+                    .check.user.properties({lang: 'tn'})
+                    .run();
+            });
+        });
+        describe("when the user selects english as language", function() {
+            it("should ask if they want to register or get info", function() {
```
Comment (smn) [inline test/personal.test.js:2069]: I think "or get info" should be dropped here

File: test/personal.test.js [hunk 2]
```diff
@@ -1843,3 +1844,303 @@ describe("app", function() {
...
+                    .check(function(api) {
+                        var metrics = api.metrics.stores.test_metric_store;
+                        assert.deepEqual(metrics['test.personal.percent_incomplete_registrations'].values, [60]);
+                        assert.deepEqual(metrics['test.personal.percent_complete_registrations'].values, [40]);
+                    })
+                    .run();
+            });
+        });
+        describe("when the user selects a different language", function() {
+            it("should ask if they want to register or get info", function() {
```
Comment (smn) [inline test/personal.test.js:2096]: same here, copy paste bug I guess.

**Comment by smn** [comment]
copy paste issues mostly, otherwise :+1:

**Pull request merged**